typescript/cli/src/tests/warp/warp-send.e2e-test.ts

```diff
@@ -130,20 +131,31 @@ describe('hyperlane warp deploy e2e tests', async function () {
       CHAIN_NAME_3,
     ]);
 
-    const ismThreshold = randomInt(1, 1e4);
+    const amountThreshold = randomInt(1, 1e4);
     const warpConfig: WarpRouteDeployConfig = {
       [CHAIN_NAME_2]: {
         type: TokenType.collateral,
         token: token.address,
         mailbox: chain2Addresses.mailbox,
         owner: ownerAddress,
+        // TODO: uncomment this once the hook logic is fixed
+        // hook: {
+        //   type: HookType.AMOUNT_ROUTING,
+        //   threshold: amountThreshold,
+        //   lowerHook: {
+        //     type: HookType.MERKLE_TREE,
```

Comment:
Can we just add an aggregation with a protocol fee hook to test that the amount routng hook does what we expect (and quote correctly